### DISCUSSION TITLE
where did you get the model for `melspectrogram.onnx`?

### DISCUSSION THREAD

**Discussion by skewballfox** [dscripka/openWakeWord#108] [Q&A]
is it being generated in the code? if so, could you point out where? I see it being loaded in `openwakeword/utils.py` but not generated. The reason I'm asking is because loading it into netron, it seems to be using a older `onnx` op set, and I wanted to see if making use of operators added to onnx after `1.13` like STFT could make a difference in speed or model size.

**Comment by dscripka** [comment]
There is an example notebook [here](https://github.com/dscripka/openWakeWord/blob/main/notebooks/converting_google_speech_embedding_model.ipynb) that shows how the melspectrogram model is defined and saved to an ONNX file.

I haven't tried it with new operators, but it's possible that it could be made faster or smaller with some experimentation.
